WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/ActivityLogListFragment.kt

```diff
@@ -76,6 +87,12 @@ class ActivityLogListFragment : Fragment() {
         } else {
             savedInstanceState.getSerializable(WordPress.SITE) as SiteModel
         }
+        val rewindableOnly = if (savedInstanceState == null) {
```

Comment:
🔍 Nitpick: I'm wondering if this `if` statement (and saving the flag in onSaveInstanceState) is needed. If the system re-creates the activity, the extras will still be available in "activity.intent", right? Could we perhaps write something like `nonNullActivity.intent?.getBooleanExtra(ACTIVITY_LOG_REWINDABLE_ONLY_KEY, false) ?: false", wdyt? 
Note: The SiteModel feels a bit different since it can change (ideally we'd store only the siteId, but that's a completely different story). However, the "rewindableOnly" key should be stable, right?

Reply:
You are totally right @malinajirka ! 🌟 

I was trying to be consistent with the `SiteModel` as you figured out yourself, but indeed this is different since the `site` instance can change.

Please expect an update soon.

PS: Just as an FYI, for me this would have been either an Idea (💡) or a Warning (⚠️), most probably the former, depending on how strict we want to be with such cases. Thank you for being kind... 🙏 
PSS: Feel free to be as strict as you need with me. ❤️

Reply:
This is now done: 67dfde26c01b334562345df27f9684a8e8d742d6

PS: Also I have update the way this happens on the activity level to reduce boilerplate code: 4594b313da84887864168de0cf37d4b1fb77d7bc

Reply:
Awesome thanks! ;)

> Just as an FYI, for me this would have been either an Idea (💡) or a Warning (⚠️)
Thanks for the info! I shared it as nitpick since it's quite common to use it the way you did in the codebase. However, I consider it as a one of many small improvements we can make when working on new code.